ui/src/pages/SiteUsersPage/SiteUserOrgItems.js

```diff
@@ -11,12 +11,11 @@ const enhance = compose(
   }),
   withProps(({ user }) => {
     const schema = 'organisation';
-    const organisations = user
-      .get('organisations')
-      .map(org => new Map({ $oid: org }));
+    const organisations = user.get('organisations').map(org => new Map({ $oid: org }));
+    // TODO: https://github.com/LearningLocker/learninglocker/pull/1513#issuecomment-587064642
```

Comment:
I don't think this is a todo, just a useful link, so let's remove the TODO part of this comment and change it to "Decision made via"